mesonbuild/scripts/gtkdochelper.py

```diff
@@ -76,7 +77,7 @@ def build_gtkdoc(source_root, build_root, doc_subdir, src_subdir,
     subprocess.check_call(mkhtml_cmd, cwd=os.path.join(abs_out, 'html'), shell=False)
     fixref_cmd = ['gtkdoc-fixxref',
                   '--module=' + module,
-                  '--module-dir=html']
+                  '--module-dir=html'] + fixxref_args
```

Comment:
No, I'm appending `fixxref_args`, which is an array resulting from `option.fixxrefargs.split('@@')`, to an array or arguments.